automated install/basic-install.sh

```diff
@@ -2026,6 +2026,9 @@ accountForRefactor() {
         fi
         echo -e "INSTALL_WEB_SERVER=$webserver_installed" >> "${setupVars}"
     fi
+
+    # Move any existing `pihole*` logs from `/var/log` to `/var/log/pihole`
+    mv /var/log/pihole*.* /var/log/pihole/ 2>/dev/null
```

Comment:
Should this be a no-clobber move? Or should we backup the destination files (if they exist)?

Maybe:
`mv --backup=simple --update --target-directory=/var/log/pihole/ /var/log/pihole*.*`

Reply:
This is the only item I have to check over before merging all of the PRs.

Reply:
I don't think we need to change this.

`-backup` is

> make a backup of each existing destination file

There shouldn't be anything in this folder, so we don't need this.

`--update` is

> move  only  when  the  SOURCE file is newer than the destination file or when the destination file is missing

So not needed, either.

Reply:
What we really want is a real `move` here, not a copy. It ensures the open file handled in `pihole-FTL` are transparently moving over as well and Pi-hole will stay able to log.

Reply:
My goals are:

1. Don't clobber existing files without a backup in case moving is not wanted or fails.
2. Don't delete the existing source files without backup in case moving is not wanted or fails.

Reply:
`pihole-FTL` should not be running when this happens, this is the installer so either this is a fresh install or someone is repairing.

>There shouldn't be anything in this folder.

True, but you know there will be.